pkg/splunk/enterprise/licensemanager.go

```diff
@@ -174,9 +175,17 @@ func ApplyLicenseManager(ctx context.Context, client splcommon.ControllerClient,
 		result = *finalResult
 
 		// trigger ClusterManager reconcile by changing the splunk/image-tag annotation
-		err = changeClusterManagerAnnotations(ctx, client, cr)
+		//err = changeClusterManagerAnnotations(ctx, client, cr)
```

Comment:
Are we removing this?

Reply:
this was mistake, i thought annotations are commented for now